DFC.EventGridSubscriptions.Data/DeadLetterEvent.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace DFC.EventGridSubscriptions.Data
+{
+    [ExcludeFromCodeCoverage]
+    public class DeadLetterEvent
+    {
+        //
+        // Summary:
```

Comment:
Might as well use XML documentation comments (```///```), so that the summaries are displayed by intellisense.

Reply:
Good point, lifted this from the MS sample